modules/AOP_Case_Updates/metadata/metafiles.php

```diff
@@ -19,7 +19,7 @@
  * or write to the Free Software Foundation,Inc., 51 Franklin Street,
  * Fifth Floor, Boston, MA 02110-1301  USA
  *
- * @author Salesagility Ltd <support@salesagility.com>
+ * @author SalesAgility Ltd <support@salesagility.com>
```

Comment:
There seems to be qutie a few like these.

Reply:
SalesAgility is a brand fix from "Salesagility". 
The coding standards has fixed that too in the 3rd line, and thats why it was also fixed here. https://docs.suitecrm.com/community/contributing-code/coding-standards/